dcr/scripts/test-vm/harvest.sh

```diff
@@ -11,6 +11,8 @@
 set -euxo pipefail
 
 ssh -o "StrictHostKeyChecking no" "$1"@"$2" "sudo tar --exclude='journal/*' --exclude='omsbundle' --exclude='omsagent' --exclude='mdsd' --exclude='scx*' --exclude='*.so' --exclude='*__LinuxDiagnostic__*' --exclude='*.zip' --exclude='*.deb' --exclude='*.rpm' -czf logs-$2.tgz /var/log /var/lib/waagent/ /etc/waagent.conf"
+# Some distros do not have "other" permissions (e.g., mariner1.0), add them here.
+ssh -o "StrictHostKeyChecking no" "$1"@"$2" "sudo chmod o+rwx logs-$2.tgz"
```

Comment:
why do we need the "other" permission

Reply:
we are creating the file using sudo, so the owner would be root... maybe the call to harvest is missing "sudo"?

Reply:
Yep, I think you're right; the difference is that mariner defaults new files to more restrictive permissions than other distros. (The call to copy the tar over is the last line of this file).

The equivalent of adding `sudo` to the harvest command would be using `root` as the user instead of `dcr` in the `scp` command. I think using the root user directly would be less ideal than this `chmod`, though.

Reply:
ah, ok, then $1 is the user... yes, probably the chown would be slightly better than a blanket all permissions to all users